Gentoo bug 337297
Link: http://bugs.gentoo.org/show_bug.cgi?id=337297
Patch-By: Jaco Kroon
Committed-Date: 2010-09-16
Ebuild-Version: 1.6.2.13-r1

Upstream bug 17991
Link: https://issues.asterisk.org/view.php?id=17991
Patch-By: Jaco Kroon
Committed-Date: none

diff -uNr asterisk-1.8.0.ORIG/apps/app_dahdiras.c asterisk-1.8.0/apps/app_dahdiras.c
--- asterisk-1.8.0.ORIG/apps/app_dahdiras.c	2010-10-27 00:41:40.000000000 +0100
+++ asterisk-1.8.0/apps/app_dahdiras.c	2010-10-27 00:50:00.000000000 +0100
@@ -119,8 +119,6 @@
 		c = strsep(&stringp, ",");
 	}
 
-	argv[argc++] = "plugin";
-	argv[argc++] = "dahdi.so";
 	argv[argc++] = "stdin";
 
 	/* Finally launch PPP */
